tools/codegen/dest/register_dispatch_key.py

```diff
@@ -43,7 +43,9 @@ def gen_registration_headers(
     elif per_operator_headers:
         headers += [
             "#include <ATen/ops/empty.h>",
-            "#include <ATen/ops/empty_strided.h>"]
+            "#include <ATen/ops/empty_strided.h>",
+            "#include <ATen/ops/_copy_from_and_resize.h>",
```

Comment:
I don't fully understand how `per_operator_headers` works, so just curious why we need to add the extra headers here?

Reply:
tldr: we used to get away with `#include <ATen/Functions.h>`, which would include every tensor operator (easy for development, but bloats all of our C++ files). The per_operator_headers stuff basically generates a new header file for every aten operator, and enforces that you only include the headers for the operators that you actually need.

The codegen uses the metadata in `native_functions.yaml` to figure out which #includes to stamp out, but there are a few ops that are hardcoded directly into the `RegisterDispatchKey.cpp` template file (that we need to hardcode here, because the codegen would know to include them just based on the yaml).

My guess for why we needed that extra `_copy_from_and_resize` op is because the LTC codegen uses it directly (to handle inplace/out= ops)